docs/advanced-topics/gke-autopilot.asciidoc

```diff
@@ -0,0 +1,259 @@
+:page_id: autopilot 
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+[id="{p}-{page_id}"]
+= Deploy ECK on GKE Autopilot
+
+This page shows how to run ECK on GKE Autopilot.
+
+* <<{p}-autopilot-deploy-the-operator>>
+* <<{p}-autopilot-deploy-elasticsearch>>
+* <<{p}-autopilot-deploy-kibana>>
+* <<{p}-autopilot-deploy-agent>>
+
+[float]
+[id="{p}-autopilot-setting-virtual-memory"]
+== Ensuring virtual memory kernel settings
+
+If you are intending to run production workloads on GKE Autopilot then `vm.max_map_count` should be set. The only way to set this kernal setting on the Autopilot hosts is with a `Daemonset` as described in the <<{p}-virtual-memory>> section. You must be running at least version 1.25 when on the `regular` channel or using the `rapid` channel, which currently runs version 1.26.
+
+CAUTION: Only use the provided `Daemonset` exactly as specified or it could be rejected by the Autopilot control plane.
+
+[id="{p}-autopilot-deploy-the-operator"]
+== Deploying the ECK Operator
+
+. Refer to <<{p}-installing-eck>> for more information on installation options.
+
+. Create a namespace to hold the Elastic resources ({eck_resources_list}):
```

Comment:
Why are we including instructions to install the operator here that are identical to our regular instructions? What is the purpose of the instruction to create a dedicated namespace? is this an Autopilot requirement (I assume not)

Reply:
> I'm following our example laid out in openshift.asciidoc where we also discuss installing the operator.

I think OpenShift is bit "special" in the sense that OpenShift users always create a "project" using `oc new-project` before creating other resources. That's why I added the `oc new-project elastic` in the case of OpenShift.

Reply:
Removed all references to creating a namespace and defaulted to the default namespace.